guide/13-managing-arcgis-applications/hub-basic-guide.ipynb

```diff
@@ -0,0 +1,675 @@
+{
```

Comment:
<p>First, we will sign-in to an organization with Hub Basic capabilities for the purpose of demonstrating these workflows.  </p><p>Note: In order to replicate these workflows, you may have to sign-in using your own credentials for your ArcGIS organization. This notebook is only designed for demonstration purposes. You may also want to verify the features available to you, per your ArcGIS Hub licensing.  </p><p>Below are examples for working with Initiatives and Events using <strong>ArcGIS Premium</strong> (insert live URL), site and page layouts, and theme editing using <strong>Enterprise Sites</strong> (insert live URL). </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Esri/arcgis-python-api/pull/1449/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='3'/>